src/components/Navbar/index.js

```diff
@@ -6,7 +6,6 @@ require('./main.css')
 const { Search } = require('../Search/')
 
 type NavbarState = {
```

Comment:
Shall we completely eliminate the NavbarState?

Reply:
we actually can't. flow needs _some_ type to parameterize that component with.

Reply:
Oooo, okay!